Examples/Example-iOS-ObjC/AppDelegate.m

```diff
@@ -27,7 +27,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                                                    reportsPerMin: 3
                                                    allowsAttachingDebugger: TRUE
                                                    detectOOM: FALSE];
-    [configuration enableBreadCrumbs:@[@(BacktraceBreadcrumbTypeManual), @(BacktraceBreadcrumbTypeLog)]];
+    [configuration enableBreadcrumbs:@[@(BacktraceBreadcrumbTypeManual), @(BacktraceBreadcrumbTypeLog)]];
```

Comment:
This code doesn't work anymore, need to be adjusted to new API

Reply:
Code updated